templates/details_base.html

```diff
@@ -64,10 +64,10 @@ <h2 class="govuk-heading-s govuk-!-margin-top-3">
                 {{entity.created | date:"jS F Y"}} ({{entity.created|naturaltime}})
               </li>
             {% endif %}
-            {% if entity.custom_properties.data_summary.last_updated %}
+            {% if entity.data_last_modified and entity.platform.urn != PlatformUrns.CADET %}
               <li>
                 <span class="govuk-!-font-weight-bold">{% translate "Last updated:" %}</span>
-                {{entity.custom_properties.data_summary.last_updated}}
+                {{entity.data_last_modified | date:"jS F Y"}}
```

Comment:
this list item should be commented out in the base and only exposed for charts while we wait on the run results being reliable